bundle/python/transform.go

```diff
@@ -0,0 +1,106 @@
+package python
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/artifacts"
+	"github.com/databricks/cli/bundle/libraries"
+	"github.com/databricks/databricks-sdk-go/service/compute"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+)
+
+// This mutator takes the wheel task and trasnforms it into notebook
+// which installs uploaded wheels using %pip and then calling corresponding
+// entry point.
+func TransformWheelTask() bundle.Mutator {
+	return &transform{}
+}
+
+type transform struct {
+}
+
+func (m *transform) Name() string {
+	return "python.TransformWheelTask"
+}
+
+const INSTALL_WHEEL_CODE = `%%pip install --force-reinstall %s`
+
+const NOTEBOOK_CODE = `
+%%python
+%s
+
+from contextlib import redirect_stdout
+import io
+import sys
+sys.argv = [%s]
+
+import pkg_resources
+_func = pkg_resources.load_entry_point("%s", "console_scripts", "%s")
```

Comment:
Instead of `console_scripts`, shouldn't this be the entry point configured in the task?

Reply:
@pietern ideally - yes. At the moment we can accept this as limitation and require users to provide entry points under `console_scripts` similar to how DBX did this.